### PR TITLE
1.0 release candidate 4

### PR DESCRIPTION
**Pull request by pthorin**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.vscode
```
Comment (perliedman) [inline .gitignore]: I would recommend against putting workflow specific files into `.gitignore`; see http://stackoverflow.com/a/1753078/890 for tips on using `.git/info/exclude` to ignore user specific files.

File: examples/local-projection-1.0.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "maxzoom": 14,
+    "crs": "EPSG:3006",
+    "projection": "+proj=utm +zone=33 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs ",
+    "transform": [1, 0, -1, 0],
+    "resolutions": [
+			8192, 4096, 2048, 1024, 512, 256, 128,
+			64, 32, 16, 8, 4, 2, 1, 0.5
+		],
+    "origin": [0, 0],
+	"bounds": [218128.7031, 6126002.9379, 1083427.2970, 7692850.9468],
```
Comment (perliedman) [inline examples/local-projection-1.0.js]: Incorrect indentation - also, this is projected coordinate bounds, not WGS84 bounds; compare to OSM bounds set in tilejson-spec here: https://github.com/mapbox/tilejson-spec/blob/master/2.1.0/example/osm.layer#L15.

In the extension I wrote, that leaflet-tilejson is modelled after, I specifically added the property `projected_bounds` for this purpose and avoid confusion of `bounds` coordinate system.

File: index.js [hunk 3]
```diff
@@ -36,6 +36,15 @@
             context.map.center = new L.LatLng(center[1], center[0]);
             context.map.zoom = center[2];
         },
+        bounds: function(context, b) {
+            // TileJson order is lng lat lng lat
+            // left, bottom, right, top
+            var left = b[0], bottom = b[1], right = b[2], top = b[3];
+            context.bounds = L.latLngBounds([bottom, left], [top, right]);
```
Comment (perliedman) [inline index.js]: This looks correct, `L.LatLngBounds` should be used here, but from the example we would pass projected coordinates in here, if I understand correctly, which would not be correct.

Leaflet's CRS object now has a [`getProjectedBounds`](http://leafletjs.com/reference-1.0.3.html#crs-getprojectedbounds) method which should return the projected bounds if specified - I don't remember if Proj4Leaflet supports that, though.

**Review by perliedman — COMMENTED** [review]
Not surprisingly, I had a few comments about `bounds` vs `projected_bounds`, that I believe should be sorted out before merging this.

File: .gitignore (hunk 1 above)
Comment (pthorin) [inline .gitignore]: Excellent, didn't know about that!

File: examples/local-projection-1.0.js (hunk 2 above)
Comment (pthorin) [inline examples/local-projection-1.0.js]: Indentation is fixed, and coordinates are now translated to WGS84 bounds.

File: index.js (hunk 3 above)
Comment (pthorin) [inline index.js]: You are, of course, correct. 

I'm also unsure if Proj4Leaflet supports that, so I left it at that for now.

Comment (pthorin) [inline index.js]: I just checked, and there is a test that uses `getProjectedBounds` in Proj4Leaflet, so that should work, if needed.

**Comment by perliedman** [comment]
:+1:

**Pull request merged**